### PR TITLE
Add support for Referrer-Policy header

### LINKED ISSUE #13: Add support for "Referrer-Policy" header
The "Referrer-Policy" header is recently introduced and adds support for global restrictions on referrer leakage. It would probably be useful to add support for it here! It's recently been added to the https://securityheaders.io testing website, with a corresponding blog post [1]. Is this something you would be open to?

[1] https://scotthelme.co.uk/a-new-security-header-referrer-policy/

### PR DESCRIPTION
**Pull request by ant1441**
Referrer-Policy is a new header to allow controlling when the Referer header is sent.

See this blog for details https://scotthelme.co.uk/a-new-security-header-referrer-policy/

Fixes #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ s := secure.New(secure.Options{
     BrowserXssFilter: true, // If BrowserXssFilter is true, adds the X-XSS-Protection header with the value `1; mode=block`. Default is false.
     ContentSecurityPolicy: "default-src 'self'", // ContentSecurityPolicy allows the Content-Security-Policy header value to be set with a custom value. Default is "".
     PublicKey: `pin-sha256="base64+primary=="; pin-sha256="base64+backup=="; max-age=5184000; includeSubdomains; report-uri="https://www.example.com/hpkp-report"`, // PublicKey implements HPKP to prevent MITM attacks with forged certificates. Default is "".
+    ReferrerPolicy: "same-origin" // ReferrerPolicy allos the Referrer-Policy header with the value to be set with a custom value. Default is "".
```
Comment (unrolled) [inline README.md]: Typo `allos` should be `allows`.

**Comment by ant1441** [comment]
Oops, was too busy trying to spell Referrer-Policy correctly (not Referer!) to spot that.
Fixed now :)

**Comment by unrolled** [comment]
Thanks for putting this together!

File: secure.go [hunk 2]
```diff
@@ -56,6 +57,8 @@ type Options struct {
 	ContentSecurityPolicy string
 	// PublicKey implements HPKP to prevent MITM attacks with forged certificates. Default is "".
 	PublicKey string
+	// Referrer Policy allows sites to control when browsers will pass the Referer header to other sites. Default is "".
```
Comment (arthurwhite) [inline secure.go]: > Oops, was too busy trying to spell Referrer-Policy correctly (not Referer!) to spot that.

Busyness overflow?  :)
"Referrer" is still misspelled here.

Comment (ant1441) [inline secure.go]: My understanding is the "Referrer-Policy" HTTP header (with two 'r's) governs which information should be included in the "Referer" HTTP header (with a single 'r').
The "Referer" header is spelt with a single 'r' due to a misspelling in the original RFC, see the Wikipedia page: https://en.wikipedia.org/wiki/HTTP_referer.

I believe that my comment is correct here, referencing the two spellings for the two headers, but I could be mistaken and just can't see it.

Comment (arthurwhite) [inline secure.go]: Well, in this sense, it makes sense indeed. 👍 
I'm out.

**Pull request merged**